_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -93841,7 +93857,8 @@
 	start_active = 1
 	},
 /turf/open/floor/bluegrid{
```

Comment:
That wasn't up to me. Boxstation's kill room floors were bluegrid, and I wasn't the one who made the kill rooms originally, so I just used what Box used.